### DISCUSSION TITLE
Borderless image resolution and position option

### DISCUSSION THREAD

**Discussion by SonoSooS** [ppy/osu#18334] [Ideas] [answered]
It would be great if it was possible to set borderless mode, but with the main image still being alterable, at least in resolution.

So for example, on an 1920x1080 screen, 1280x1024 borderless screen would be centered, with the rest being filled with black, just like in fullscreen mode, except without the side effects of fullscreen mode.
It would be even better if the image position could be altered arbitarily instead of being centered, either for absolute input devices, or for image capture purposes.

1280x1024 clips into the taskbar in windowed mode, and native fullscreen has some side effects which make it unsuitable for some cases, which borderless mode doesn't experience.

**Comment by smoogipoo** [comment, marked as answer]
Isn't this the "screen scaling" option in the settings?

**Reply by peppy** [reply]
it is.

if this is about being able to choose a specific size rather than percentage, then it's being tracked in many other issues and will probably be added in the future based on demand.

**Reply by SonoSooS** [reply]
Oops, I didn't notice the sliders, because of the animation.

But yeah, being able to specify a specific value would be better than some vague percentages.
